python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -281,6 +282,42 @@ def run_individual(self, number, repeat=1, min_repeat_ms=0):
         ret = self._run_individual(number, repeat, min_repeat_ms)
         return ret.strip(",").split(",") if ret else []
 
+    def run_individual_node(self, index, number, repeat=1, min_repeat_ms=0):
```

Comment:
Number should default to the same as `time_evaluator` (10 I think).

Reply:
done